src/bbt/java/org/sonar/plugin/cayc/bbt/CAYCChartDataBBT.java

```diff
@@ -93,6 +98,33 @@ public void open_plugin_page() throws URISyntaxException {
     $(byTagAndText("a", "Clean as You Code")).click();
     $("[data-testid='current-issue-count']").shouldHave(Condition.text(issuesNumber + " issues"));
 
+    $("[aria-label='Project']").click();
+    $("#react-select-2-option-1").click();
+
+    long javaSampleIssuesNumber = wsClient
+      .issues()
+      .search(new SearchRequest().setTypes(List.of("BUG", "VULNERABILITY")).setComponentKeys(List.of("org.sonarsource.plugins.cayc:java-sample")))
+      .getIssuesCount();
+
+    $("[data-testid='current-issue-count']").shouldHave(Condition.text(javaSampleIssuesNumber + " issues"));
+
+    $("[aria-label='Project']").click();
+    $("#react-select-2-option-2").click();
+
+    long javaSample1IssuesNumber = wsClient
+      .issues()
+      .search(new SearchRequest().setTypes(List.of("BUG", "VULNERABILITY")).setComponentKeys(List.of("org.sonarsource.plugins.cayc:java-sample-1")))
+      .getIssuesCount();
+
+    $("[data-testid='current-issue-count']").shouldHave(Condition.text(javaSample1IssuesNumber + " issues"));
+  }
+
+  @Test
+  public void no_issues_message_for_no_analyzed_projects() {
```

Comment:
I'm not sure about the value of this test. This use-case is already tested at the RTL level.